src/main/kotlin/di/graphQlDi.kt

```diff
@@ -30,17 +30,14 @@ val graphQlDi = DI.Module("graphql") {
     bind<MessageMutation>() with singleton { MessageMutation(instance(), instance()) }
     bind<PushMutation>() with singleton { PushMutation(instance()) }
     bind<GraphQLSchema>() with singleton {
-        val objectMapper = jacksonObjectMapper().apply {
-            registerModule(JavaTimeModule())
-        }
         val config = SchemaGeneratorConfig(
             supportedPackages = listOf(
                 "persistence.jooq.tables.pojos",
                 "clientapi.models",
                 "models",
             ),
             hooks = ChatServerSchemaGeneratorHooks(),
-            dataFetcherFactoryProvider = SimpleKotlinDataFetcherFactoryProvider(objectMapper = objectMapper)
+            dataFetcherFactoryProvider = SimpleKotlinDataFetcherFactoryProvider()
```

Comment:
object mapper configuration is not required anymore? 🤔

Reply:
Because I saw you still have an object mapper instance and maybe graphql should use the same configuration🤔

Reply:
It is not possible to pass an object mapper anymore. I still need to test if it works, though. :D